src/drivers/mpu6000/mpu6000.cpp

```diff
@@ -1288,6 +1329,7 @@ MPU6000::ioctl(struct file *filp, int cmd, unsigned long arg)
 					float cutoff_freq_hz = _accel_filter_x.get_cutoff_freq();
 					float sample_rate = 1.0e6f / ticks;
 					_set_dlpf_filter(cutoff_freq_hz);
+					_set_icm_acc_dlpf_filter(cutoff_freq_hz);
```

Comment:
@davids5 Is this alright? This means that the reg 0x1D will also be configured on the MPU6000. It doesn't mean anything on the MPU6000 that I'm aware of. But I kinda feel I should put an if clause to only execute that when the device is an ICM